index.js

```diff
@@ -1,9 +1,9 @@
 var express = require("express");
-var http = require("http");
 var app = express();
+var dao = require('./app/dao/dao');
+var port = process.env.PORT || 8000;
+require('./app/routes/routes')(app, dao);
```

Comment:
I don't understand what (app, dao) this mean?

Reply:
I need to pass an instance of express and dao to populate it with routings and to use the dao.